lms/envs/bok_choy.py

```diff
@@ -237,8 +237,6 @@
 from waffle.models import Flag
 try:
     flag, created = Flag.objects.get_or_create(name='unified_course_view')
-    flag.everyone = True
-    flag.save
```

Comment:
REVIEWERS: @nedbat pointed out that flag.save wasn't calling save(), and this code isn't needed.  What was needed here will remain, which is:
1. The creation of the flag above, and
2. The setting to allow WAFFLE_OVERRIDE below